requirements.txt

```diff
@@ -1,7 +1,3 @@
-requests~=2.30.0
-setuptools~=41.0.1
-websocket-client~=0.58.0
-
 # Documentation dependencies
 Sphinx~=7.1.0
```

Comment:
you should also add these as `[docs]` in `optional-dependencies`

Reply:
Good idea! Done - the toml syntax is really cool